website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,122 @@
+<link rel="stylesheet" href='/static/css/pages/remove-contributor-page.css'>
+<div id="removeContributor" class="modal fade">
+    <div data-bind="css: modalSize">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+            <div class="modal-body" >
+                <div data-bind="if: canRemoveNode() && !pageChanged()">
+                    <!-- remove page -->
+                    <div data-bind='if:page() === REMOVE'>
+                        <div class="form-group">
+                            <span>Do you want to remove <b>{{contributorToRemove()["fullname"]}}</b> from <b>{{title}}</b>, or from <b>{{title}}</b> and every component in it.</span>
```

Comment:
There should be a question mark at the end instead of a period.

![screen shot 2016-01-12 at 3 05 51 pm](https://cloud.githubusercontent.com/assets/134455/12275406/fe7b4922-b93d-11e5-8295-0e956e69e229.png)